BadApi/Controllers/PostController.cs

```diff
@@ -0,0 +1,100 @@
+﻿using BadApi.Controllers.Base;
+using BadApi.Domain;
+using BadApi.Models.Post;
+using Dapper;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using System.Web.Http;
+
+namespace BadApi.Controllers
+{
+	[RoutePrefix("api/post")]
+    public class PostController : BaseApiController
+    {
+		private static List<ViewPostModel> _postCache = new List<ViewPostModel>();
```

Comment:
Cache should be stored in a thread-safe collection and implement the ICache interface - this should be in a different class and made generic for consumption from all controllers